### PR TITLE
feat(route): add google research blog

### PR DESCRIPTION
**Pull request by Levix**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/google/research
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [x] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/routes/google/research.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+export const route: Route = {
+    path: '/research',
+    categories: ['blog'],
+    example: '/research',
+    name: 'Google Research Blog',
+    maintainers: ['Levix'],
+    radar: [
+        {
+            source: [baseUrl],
```
Comment (TonyRL) [inline lib/routes/google/research.ts:16]: The value of source should be: `['research.google']`

File: lib/routes/google/research.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+
+const baseUrl = `https://research.google`;
+
+export const route: Route = {
+    path: '/research',
+    categories: ['blog'],
+    example: '/research',
```
Comment (TonyRL) [inline lib/routes/google/research.ts:11]:
```suggestion
    example: '/google/research',
```

File: lib/routes/google/research.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+
+const baseUrl = `https://research.google`;
```
Comment (TonyRL) [inline lib/routes/google/research.ts:6]:
```suggestion
const baseUrl = 'https://research.google';
```

File: lib/routes/google/research.ts [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+                    author: 'Google',
+                    category: item
+                        .find('.not-glue.caption')
+                        .toArray()
+                        .map((item) => $(item).text().replace('·', '').trim()),
+                };
+            });
+
+        return {
+            title: `Google Research Blog`,
```
Comment (TonyRL) [inline lib/routes/google/research.ts:40]:
```suggestion
            title: 'Google Research Blog',
```

File: lib/routes/google/research.ts [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+
+const baseUrl = `https://research.google`;
+
+export const route: Route = {
+    path: '/research',
+    categories: ['blog'],
+    example: '/research',
+    name: 'Google Research Blog',
```
Comment (TonyRL) [inline lib/routes/google/research.ts:12]:
```suggestion
    name: 'Research Blog',
```

File: lib/routes/google/research.ts (hunk 1 above)
Comment (Levix) [reply lib/routes/google/research.ts]: fix

File: lib/routes/google/research.ts (hunk 2 above)
Comment (Levix) [reply lib/routes/google/research.ts]: fix

File: lib/routes/google/research.ts (hunk 3 above)
Comment (Levix) [reply lib/routes/google/research.ts]: fix

File: lib/routes/google/research.ts (hunk 4 above)
Comment (Levix) [reply lib/routes/google/research.ts]: fix

File: lib/routes/google/research.ts (hunk 5 above)
Comment (Levix) [reply lib/routes/google/research.ts]: fix

**Pull request merged**